docs/Updating-v0.3.md

```diff
@@ -0,0 +1,24 @@
+# Migrating from v0.2 to v0.3
+
+There area number of new features and improvements in ML-Agents v0.3 which change both the training process and Unity API in ways which will cause imcompatabilities with environments made using older versions. This page is designed to highlight those changes for users familiar with v0.2 in order to make the trainsition simple.
+
+## Important
+ * ML-Agents no longer supports usage with Python 2. 
+
+## Python Training
+ * The `learn.py` script now replaces `ppo.py` and the `PPO.ipynb` python notebook as the launching point for training with ML-Agents. For more information on using `learn.py`, see [here]().
```

Comment:
When describing changes, talk from the perspective of v0.2. Here, for example, it would say: "The training script ppo.py and PPO.ipynb Python notebook have been replaced with a single learn.py script."